go/common/rpc/converters.go

```diff
@@ -37,23 +37,13 @@ func ToBlockSubmissionResponseMsg(response *common.BlockSubmissionResponse) (gen
 	}
 
 	return generated.BlockSubmissionResponseMsg{
-		BlockHeader:             ToBlockHeaderMsg(response.BlockHeader),
 		UpdatedHeadRollup:       response.UpdatedHeadRollup,
 		RollupHead:              ToRollupHeaderMsg(response.IngestedRollupHeader),
 		SubscribedLogs:          subscribedLogBytes,
 		ProducedSecretResponses: ToSecretRespMsg(response.ProducedSecretResponses),
 	}, nil
 }
 
-func ToProduceGenesisResponseMsg(response *common.ProduceGenesisResponse) generated.ProduceGenesisResponseMsg {
```

Comment:
Lots of cruft can go now, including this wrapper (since we just return a rollup), and the converters for block headers.